Tests/ModelTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Part of the Joomla GSoC Webservices Project
+ *
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE
+ */
+
+namespace Joomla\Entity\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Joomla\Entity\Tests\Models\User;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+/**
+ * Class ModelTest
+ * @package Joomla\Entity\Tests
+ * @since 1.0
+ */
+class ModelTest extends TestCase
+{
+
+	/**
```

Comment:
For these tests to actually run we're going to need to configure the `User` database table (use DBUnit and see how the CMS does this)